### DISCUSSION TITLE
codespaces can't copy and paste on edge on mac cmd-c cmd-v doesn't do anything!!!

### DISCUSSION THREAD

**Discussion by kundeng** [community/community#123171] [Codespaces]
### Select Topic Area

Bug

### Body

Has anyone experienced this?  I tried multiple codespaces from several repositories.

**Comment by techpond31** [comment]
🤔Although Edge on Mac supports standard keyboard shortcuts for cut, copy, and paste (Cmd + X, Cmd + C, and Cmd + V respectively), there seem to be inconsistencies with Microsoft Codespaces integration.

A few users reported experiencing similar issues in the past. Most of them managed to solve the problem by changing their default keyboard shortcut preferences in Microsoft Edge.

Please follow these steps to modify the keyboard shortcut configuration in Edge:

1. Open Microsoft Edge on your Mac.
2. Click on the three dots located at the top right corner of the window to reveal the menu. Select "Preferences." Alternatively, press "Cmd + ,".
3. Scroll down until you reach the "Shortcuts" tab on the left sidebar.
4. Locate the "Cut," "Copy," and "Paste" entries listed alphabetically under "All your shortcuts."
5. Confirm that the assigned keys for Cut, Copy, and Paste are indeed set to the expected values ("X," "C," and "V").
6. If the bindings appear correct, attempt editing them by double-clicking the current keys sequence next to "Keyboard Shortcut." Input the same combinations (Cmd + X, Cmd + C, Cmd + V) manually, then hit Enter to save changes.

After modifying the keyboard shortcuts as suggested, restart your Microsoft Edge session and launch a fresh CodeSpace. Test the newly configured cut, copy, and paste commands (Cmd + X, Cmd + C, Cmd + V) inside the integrated terminal. These adjustments should restore normal functioning.

Should the issue persist, consider reporting the bug directly to Microsoft via the Feedback Hub app included with Visual Studio Code. Sharing detailed observations helps developers prioritize fixes and improve experiences for all users. Moreover, exploring alternative browsers like Chrome, Firefox, or Safari might serve as temporary workarounds until a solution becomes available.
